site/source/docs/api_reference/proxying.h.rst

```diff
@@ -72,19 +72,19 @@ Functions
 
   Signal the end of a task proxied with ``emscripten_proxy_sync_with_ctx``.
 
-.. c:function:: int emscripten_proxy_async(em_proxying_queue* q, pthread_t target_thread, void (\*func)(void*), void* arg)
+.. c:function:: int emscripten_proxy_async(em_proxying_queue* q, pthread_t target_thread, void (*func)(void*), void* arg)
 
   Enqueue ``func`` to be called with argument ``arg`` on the given queue and
   thread then return immediately without waiting for ``func`` to be executed.
   Returns 1 if the work was successfully enqueued or 0 otherwise.
 
-.. c:function:: int emscripten_proxy_sync(em_proxying_queue* q, pthread_t target_thread, void (\*func)(void*), void* arg)
+.. c:function:: int emscripten_proxy_sync(em_proxying_queue* q, pthread_t target_thread, void (*func)(void*), void* arg)
 
   Enqueue ``func`` to be called with argument ``arg`` on the given queue and
   thread then wait for ``func`` to be executed synchronously before returning.
   Returns 1 if the ``func`` was successfully completed and 0 otherwise.
 
-.. c:function:: int emscripten_proxy_sync_with_ctx(em_proxying_queue* q, pthread_t target_thread, void (\*func)(em_proxying_ctx*, void*), void* arg)
+.. c:function:: int emscripten_proxy_sync_with_ctx(em_proxying_queue* q, pthread_t target_thread, void (*func)(em_proxying_ctx*, void*), void* arg)
```

Comment:
The backslash was there to keep `func)(em_proxying_ctx` from being italicized. Was it unnecessary?

Reply:
Sphinx 2.4.4, without this PR:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/8726997/201422672-b381259e-1979-4827-acab-d05cc4093811.png">

Sphinx 2.4.4, with this PR:
<img width="717" alt="image" src="https://user-images.githubusercontent.com/8726997/201422737-6c1b32f7-a33e-4815-bed1-fb15baa26b69.png">

So apparently there are no changes in the generated html file.

Sphinx 4.5.0, without this PR: Doesn't build due to errors (which this PR fixes)

Sphinx 4.5.0, with this PR:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/8726997/201422850-91bd8f2f-a993-493e-a671-6b5954747bd3.png">
Weird that with 4.5.0, the function name colors are gone, and all function signatures are in italics. This is not due to the changes in this PR; all function signatures are displayed this way. 

I also tried to install Sphinx 5.3.0, the newest, but failed due to some other dependences in my machine.

So, while we are not updating the official required Sphinx version (I tried to allow Sphinx version higher than 2.4.4 in #18181, but was told that higher versions have problems), this PR fixes errors generated with building with higher versions of Sphinx, while doesn't affect the current version (2.4.4).

Reply:
Great, thanks for investigating!